pkg/file/path.go

```diff
@@ -72,7 +70,7 @@ func (p Path) ParentPath() (Path, error) {
 	return sanitized, nil
 }
 
-// AllPaths returns all valid constituent paths (e.g. /home/wagoodman/file.txt -> /, /home, /home/wagoodman )
+// AllPaths returns all valid constituent paths (e.g. /home/wagoodman/file.txt -> /, /home, /home/wagoodman, /home/wagoodman/file.txt )
```

Comment:
Re "constituent paths", it'd be good to figure out a more clear way to express which paths get returned. We use "all valid constituent paths" here in the documentation for `AllPaths()` and also for `ConstituentPaths()`, so it's unclear if "constituent" is meant to include the original path itself. Perhaps something like `AllPaths` and `AncestorPaths`?

Reply:
I think the description needs to be updated (it is currently wrong for both functions), I can adjust.